framework_tvb/tvb/interfaces/rest/commons/dtos.py

```diff
@@ -56,10 +56,8 @@ def __init__(self, user=None, **kwargs):
         """
         self.update(kwargs)
         if user is not None:
-            self.username = user.username
+            self.gid = user.gid
```

Comment:
We should also have display_name here, right?

Reply:
Yes. We will have also the display_name field in the DTO